practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/problem/src/Select.test.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
+//manually mocking toSelectOptions
+
+import { render } from "@testing-library/react";
+import { Select } from "./Select";
+import { toSelectOptions } from "./toSelectOptions";
+
+// mock the toSelectOptions module with a callback that returns a mock module
+//   mocking the toSelectOptions function
```

Comment:
It seems like these tests start off by passing in CodeSandbox… should they start off as failing?

Reply:
Yeah.  `.toBeDefined()` was the wrong predicate to use there.  It should be `.not.toBe(null)`.  I'll fix that.